### PR TITLE
Add checkpoints for OMZ 2019 R2 models

### PR DESCRIPTION
**Pull request by druzhkov-paul**

### CODE REVIEW

**Comment by druzhkov-paul** [comment]
@DmitriySidnev please verify.
@snosov1 @AlexanderDokuchaev please review.

**Comment by DmitriySidnev** [comment]
:+1:

File: pytorch_toolkit/instance_segmentation/OpenVINO.md [hunk 1]
```diff
@@ -41,6 +41,14 @@ $ python3 tools/train.py \
...
+
 ```bash
 $ python3 tools/train_0050.py
 ```
+
+Sample fine-tuning sample can be found at `tools/finetune_0050.py`.
+Checkpoint has to be downloaded via
+`tools/download_pretrained_weights.py` first.
+Please make necessary changes for learning rate, batch size,
+number of training step or other training parameters there before running.
```
Comment (snosov1) [inline pytorch_toolkit/instance_segmentation/OpenVINO.md]:
```suggestion
number of training steps or other training parameters there before running.
```

File: pytorch_toolkit/instance_segmentation/OpenVINO.md [hunk 2]
```diff
@@ -41,6 +41,14 @@ $ python3 tools/train.py \
 
 [Description](https://github.com/opencv/open_model_zoo/blob/develop/models/intel/instance-segmentation-security-0050/description/instance-segmentation-security-0050.md)
 
+For training from ImageNet weights run: 
+
 ```bash
 $ python3 tools/train_0050.py
 ```
+
+Sample fine-tuning sample can be found at `tools/finetune_0050.py`.
```
Comment (snosov1) [inline pytorch_toolkit/instance_segmentation/OpenVINO.md]:
```suggestion
Fine-tuning sample can be found at `tools/finetune_0050.py`.
```

Comment (druzhkov-paul) [reply pytorch_toolkit/instance_segmentation/OpenVINO.md]: Fixed.

File: pytorch_toolkit/instance_segmentation/OpenVINO.md (hunk 1 above)
Comment (druzhkov-paul) [reply pytorch_toolkit/instance_segmentation/OpenVINO.md]: Fixed.

**Pull request merged**